### ISSUE TITLE
Question about the training loss in code

### ISSUE THREAD

**Issue by SuperHan777** [omni-us/research-GANwriting#15]
Thanks for releasing the code of your awesome paper. I have a question about the loss calculation in network_pro.py. You generate a new text using the new_ed1 function in load_data.py and use the generated image of the new text string to calculate the discriminator loss, writer classifier loss and the recognition loss together with the generated image of the target text string. What is the purpose of this processing?
